db/models.py

```diff
@@ -1,10 +1,12 @@
 from django.db import models
 
+from django.forms import CharField
```

Comment:
It seems unnecessary to import `CharField` from `django.forms` as it is not used in this file. Consider removing this import to clean up the code.